platform/spark/modules/EEPROM/eGuiSettings.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2015 BrewPi/Elco Jacobs/Matthew McGowan.
+ *
+ * This file is part of BrewPi.
+ * 
+ * BrewPi is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * BrewPi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with BrewPi.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef EGUISETTINGS_H
+#define	EGUISETTINGS_H
+
+extern "C" {
+#include "d4d.h"
+}
+
+#include "flashee-eeprom.h"
+#include "SparkEepromRegions.h"
+
+
+// Reserve 32 bytes for touch calibration settings to have room for growth
+union calibrationSettingsArea {
+    D4D_TOUCHSCREEN_CALIB touchCalib;
+    unsigned char reserved[32];
```

Comment:
This does no harm, but also not necessary. The flashee region was several K in size (32 iirc)